docs/src/docs/next-steps.md

```diff
@@ -21,13 +65,17 @@ If you need a quick way to sign a ModelKit you can follow the same instructions
 
 ## Making your own Kitfile
 
-A Kitfile is the configuration document for your ModelKit. It's written in YAML so it's easy to read. There are five main parts to a Kitfile:
+A Kitfile is the configuration document for your ModelKit. It's similar to a recipe or a `dockerfile` for a container. It's written in YAML so it's easy to read. 
```

Comment:
I completely forgot to actually document this, but there is also the `kit init` command to get started with a Kitfile in any directory.